packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -439,7 +439,105 @@ void main() {
 
     if (isTargetPlatformiOS) {
       await tester.tap(find.text('Search Web'));
-      expect(lastLookUp, 'Test');
+      expect(lastSearch, 'Test');
+    }
+  },
+    variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS, TargetPlatform.android }),
+    skip: isContextMenuProvidedByPlatform, // [intended] only applies to platforms where we supply the context menu.
+  );
+
+  testWidgets('Share shows up on iOS only (CupertinoTextField)', (WidgetTester tester) async {
```

Comment:
does share show up in material for ios?

Reply:
no (?), looks like cupertino to me

Reply:
Regardless of whether TextField or CupertinoTextField is being used, the default context menu will reflect the current platform.  So if you use TextField on an iPhone, you'll still get the iOS context menu.

So this test and the following test look correct as written to me.  The only thing I would change is to move the TextField test below to material/text_field_test.dart.